### PR TITLE
feat(ecs-patterns): LoadBalancedFargateService - allow specifying containerName and role

### PR DESCRIPTION
**Pull request by DavidChristiansen**
Implementation of execution tests, task role and container name overrides + tests

----

### Pull Request Checklist

* [x] Testing
  - Unit test added (prefer not to modify an existing test, otherwise, it's probably a breaking change)
  - __CLI change?:__ coordinate update of integration tests with team
  - __cdk-init template change?:__ coordinated update of integration tests with team
* [x] Docs
  - __jsdocs__: All public APIs documented
  - __README__: README and/or documentation topic updated
  - __Design__: For significant features, design document added to `design` folder
* [x] Title and Description
  - __Change type__: title prefixed with **fix**, **feat** and module name in parens, which will appear in changelog
  - __Title__: use lower-case and doesn't end with a period
  - __Breaking?__: last paragraph: "BREAKING CHANGE: <describe what changed + link for details>"
  - __Issues__: Indicate issues fixed via: "**Fixes #xxx**" or "**Closes #xxx**"
* [x] Sensitive Modules (requires 2 PR approvers)
  - IAM Policy Document (in @aws-cdk/aws-iam)
  - EC2 Security Groups and ACLs (in @aws-cdk/aws-ec2)
  - Grant APIs (only if not based on official documentation with a reference)

----

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license.

### CODE REVIEW

File: packages/@aws-cdk/aws-ecs-patterns/lib/fargate/load-balanced-fargate-service.ts [hunk 1]
```diff
@@ -72,6 +73,27 @@ export interface LoadBalancedFargateServiceProps extends LoadBalancedServiceBase
    * @default true
    */
   readonly createLogs?: boolean;
+
+  /**
+   * Override for the Fargate Task Definition execution role
+   *
+   * @default - No value
```
Comment (eladb) [inline packages/@aws-cdk/aws-ecs-patterns/lib/fargate/load-balanced-fargate-service.ts]: Describe the default *behavior*, not the default value

File: packages/@aws-cdk/aws-ecs-patterns/lib/fargate/load-balanced-fargate-service.ts [hunk 2]
```diff
@@ -72,6 +73,27 @@ export interface LoadBalancedFargateServiceProps extends LoadBalancedServiceBase
...
+   * Override for the Fargate Task Definition task role
+   *
+   * @default - No value
+   */
+  readonly taskRole?: iam.Role;
+
+  /**
+   * Override value for the container name
+   *
+   * @default - No value
```
Comment (eladb) [inline packages/@aws-cdk/aws-ecs-patterns/lib/fargate/load-balanced-fargate-service.ts]: The default is "web" I believe

File: packages/@aws-cdk/aws-ecs-patterns/lib/fargate/load-balanced-fargate-service.ts (hunk 1 above)
Comment (DavidChristiansen) [reply packages/@aws-cdk/aws-ecs-patterns/lib/fargate/load-balanced-fargate-service.ts]: Fixed

File: packages/@aws-cdk/aws-ecs-patterns/lib/fargate/load-balanced-fargate-service.ts (hunk 2 above)
Comment (DavidChristiansen) [reply packages/@aws-cdk/aws-ecs-patterns/lib/fargate/load-balanced-fargate-service.ts]: Fixed

File: packages/@aws-cdk/aws-ecs-patterns/test/fargate/test.load-balanced-fargate-service.ts [hunk 3]
```diff
@@ -48,5 +49,138 @@ export = {
...
+      image: ecs.ContainerImage.fromRegistry("/aws/aws-example-app"),
+    });
+
+    // THEN
+    const serviceTaskDefinition = SynthUtils.synthesize(stack).template.Resources.ServiceTaskDef1922A00F;
+    test.deepEqual(serviceTaskDefinition.Properties.ContainerDefinitions[0].Name, 'web');
+    test.done();
+  },
+
+  // 'setting servicename updates service name with given name'(test: Test) {
```
Comment (leepa) [inline packages/@aws-cdk/aws-ecs-patterns/test/fargate/test.load-balanced-fargate-service.ts]: Commented out code with no reason for existence.

**Pull request merged**